net/Cargo.toml

```diff
@@ -8,17 +8,14 @@ license = "Apache-2.0"
 [features]
 default = []
 serde = ["dep:serde"]
-arbitrary = ["dep:arbitrary", "dep:bolero"]
+arbitrary = ["dep:bolero"]
 test_buffer = []
 
 [dependencies]
 ahash = { workspace = true }
-arbitrary = { workspace = true, features = ["derive"], optional = true }
-arc-swap = { workspace = true }
 arrayvec = { workspace = true }
 bolero = { workspace = true, features = ["alloc", "arbitrary", "std"], optional = true }
 derive_builder = { workspace = true, features = [] }
-dyn-iter = { workspace = true }
```

Comment:
I think these were introduced in this PR, what were they needed for in the first place?

Reply:
I think dyn-iter is a leftover when pipeline was in dataplane. 

I've just removed as part of rebasing https://github.com/githedgehog/dataplane/pull/377  on the latest main, which is ready to go.